components/molecule/accordion/src/index.scss

```diff
@@ -0,0 +1,117 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+@import './settings';
+
+.sui-MoleculeAccordion-tab {
+  background-color: $bgc-accordion-tab-container;
+  border: $bd-accordion-tab-container;
+  border-bottom: $bd-accordion-tab-container-none;
+  color: $c-accordion-tab-text;
+  display: flex;
+  flex-direction: column;
+  font-size: $fz-base;
+  margin: 0;
+  padding: 0;
+  position: relative;
+  width: 100%;
+
+  &:last-child {
+    border-bottom: $bd-accordion-tab-container;
+  }
+
+  &Btn {
+    background: none;
+    border: 0;
+    color: $c-accordion-tab-text;
+    cursor: pointer;
+    font-size: $fz-base;
+    height: $h-accordion-tab-btn;
```

Comment:
do you think it's necessary?

Reply:
i mean, i usually try to avoid using fixed heights. i prefer to let content expand the container height

Reply:
The height is delimited in the paper, so i forced it to be always 40px. Would you make it adaptative to the title font-size?

Reply:
Yes, me too. But in this case as it was defined in the paper I thought it should be fix

Reply:
oh, i see.
nothing to say then. 
Well done mate! 😊